extensions/amp-story/1.0/test/test-amp-story-bookend.js

```diff
@@ -957,6 +967,47 @@ describes.fakeWin('amp-story-bookend', {win: {location}, amp: true}, (env) => {
         'https://www.nationalgeographic.com/amp-stories/assets/01-iconic-american-destinations.jpg'
       );
     });
+
+    it('should have an alt attribute on the image', () => {
+      const noAltComponent = {
+        url: 'http://example.com/small.html',
+        domainName: 'example.com',
+        type: 'small',
+        title: 'This image has no alt attribute',
+        image: 'http://placehold.it/256x128',
+      };
+
+      const emptyAltComponent = {
+        url: 'http://example.com/small.html',
+        domainName: 'example.com',
+        type: 'small',
+        title: 'This image has an empty alt attribute',
+        image: 'http://placehold.it/256x128',
+        alt: '',
+      };
+
+      const textAltComponent = {
+        url: 'http://example.com/small.html',
+        domainName: 'example.com',
+        type: 'small',
+        title: 'This image has text in its alt attribute',
+        image: 'http://placehold.it/256x128',
+        alt: 'test',
+      };
+
+      const small = new ArticleComponent();
+
+      const noAlt = small.buildElement(noAltComponent, win, {position: 0});
+      expect(noAlt.querySelector('img').alt).to.equal('');
+
+      const emptyAlt = small.buildElement(emptyAltComponent, win, {
+        position: 0,
+      });
+      expect(emptyAlt.querySelector('img').alt).to.equal('');
+
+      const textAlt = small.buildElement(textAltComponent, win, {position: 0});
+      expect(textAlt.querySelector('img').alt).to.equal('test');
```

Comment:
Tests need to be focused on one concept. One `expect` per test.
Please separate these.

They could be something like:
`'should have an empty alt attribute on image if no alt is specified'`
`'should have an empty alt attribute on image if empty alt is specified'`
`'should propagate alt attribute if specified'`

I'm not sure that we need to test if the alt is empty if it was specified as empty. 
Do you think the first test covers that?

Reply:
I think you are right, testing alt is empty if it was specified empty seems redundant.  I'll separate these tests!